.azure/hpu-tests.yml

```diff
@@ -20,23 +20,44 @@ jobs:
     timeoutInMinutes: "10"
     # how much time to give 'run always even if cancelled tasks' before stopping them
     cancelTimeoutInMinutes: "2"
-    pool: intel-hpus
+    pool: habana-gaudi-hpus
```

Comment:
nit: `hpus` has Habana term in it

Reply:
well, yes, but I still wanted to preserve it as all others have it...
I there are temporary two pools so we can rename it later :rabbit: